src/python_interpreter.rs

```diff
@@ -22,6 +22,10 @@ print(json.dumps({
     "major": sys.version_info.major,
     "minor": sys.version_info.minor,
     "abiflags": sysconfig.get_config_var("ABIFLAGS"),
+    # implementation_lower is only specified by pypy
+    "interpreter": sysconfig.get_config_var("implementation_lower") or "cpython",
```

Comment:
I'd prefer `platform.python_implementation()` because that part of the official python docs.